### DISCUSSION TITLE
How to change location of "blockchain_v1_mainnet.sqlite" file?

### DISCUSSION THREAD

**Discussion by dial-888** [Chia-Network/chia-blockchain#9684] [Q&A]
How to change location of "blockchain_v1_mainnet.sqlite" file from \.chia\mainnet\db ? It take 40GB on C:\ drive. But I can't find this file in "config.yaml" to change the path.

**Comment by chinabala** [comment]
If it's on Windows, then using CMD terminal:
mklink /J C:\xxxx\.chia\mainnet\db D:\folder_you_want
